datasets/norwegian_ner/README.md

```diff
@@ -0,0 +1,137 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- sv
+licenses:
+- other-
```

Comment:
is this other or unknown ? please use the tagger to use valid license types

Reply:
Haha good catch ! Pinging @jplu

Reply:
Good catch indeed 😄 Taking care of this!